bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/handler/DaikinBaseHandler.java

```diff
@@ -92,6 +92,10 @@ public DaikinBaseHandler(Thing thing, DaikinDynamicStateDescriptionProvider stat
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
+        if (webTargets == null) {
+            logger.error("Unable to handle commands. Check Daikin binding configurations");
```

Comment:
Rather WARN level

Reply:
if `webTargets` is null the binding is not going to work at all. Shouldn't that warrant an ERROR level? It is an exceptional condition that shouldn't be encountered in the normal operation, even when the binding has been misconfigured, this condition should still not occur. I don't even know how to cause webTargets to become null, if OSGI works properly.

Reply:
However, the log message should be changed to clarify the situation though

Reply:
I've changed this to warn. It seems that I should never use `logger.error` in any case, within a binding.